web-app/src/smart-locks/components/smart-lock-list-item.component.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import DoorsIcon from "../../shared/icons/doors-icon.component.vue";
+
+const props = defineProps<{
+  name: string;
+  keysCount: number;
+  id: number;
+}>();
+
+const emit = defineEmits<{
+  (e: "manage", id: number): void;
+}>();
+
+const onClick = () => {
+  emit("manage", props.id);
+};
+</script>
+
+<template>
+  <div
+    @click="onClick"
+    class="bg-white rounded-large h-36 w-64 m-4 p-5 cursor-pointer"
+  >
```

Comment:
This seems to be an element that the user can click on, so consider using a `button` tag instead of a `div`. This way, all the browser logic and accessibilty feature (such as screen reader support and keyboard navigation) is automatically available